layouts/partials/scientific-domains.html

```diff
@@ -55,7 +55,7 @@
             <td class="center-text"><a href="https://github.com/mwaskom/seaborn">Seaborn</a></td>
             <td class="center-text"></td>
             <td class="center-text"></td>
-            <td class="center-text"></td>
+            <td class="center-text"><a href="https://github.com/pyvista/pyvista">PyVista</a></td>
```

Comment:
I figured I'd put the git repository here as landing pages and docs are subject to change, downtime, and other variability. The github repo will always be there though

Reply:
Actually I missed this line, I'd prefer to keep PyVista in the Visualization tab only, because it doesn't seem to be an image processing tool as meant in this category. 

There's a merge conflict anyway I'm resolving, I'll take out this line before merging.